plugins/node/opentelemetry-instrumentation-pg/README.md

```diff
@@ -0,0 +1,75 @@
+# OpenTelemetry Postgres Instrumentation for Node.js
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`pg and pg-pool`](https://github.com/brianc/node-postgres)
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-pg
+```
+
+It will install Instrumentation for PG and PG-POOL
+
+## Usage
+
+To load all of the [default supported plugins](https://github.com/open-telemetry/opentelemetry-js#plugins), use the below approach. Each plugin is only loaded when the module that it patches is loaded; in other words, there is no computational overhead for listing plugins for unused modules.
+
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const provider = new NodeTracerProvider();
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+registerInstrumentations();
```

Comment:
```suggestion
registerInstrumentations({
  instrumentations: new PostgresInstrumentation()
});
```
And add `const { PostgresInstrumentation } = require('@opentelemetry/instrumentation-pg');` above.

At least this is how I understand the `registerInstrumentations()` API - it requires an argument.

Reply:
Hmm, the options should be optional, I will add `{}` for now. But please read the whole context, this is just showing how you will normally load the default plugins now, not the instrumentations. Then you have next paragraph where it is shown how to load `PostgresInstrumentation`

Reply:
For me this is confusing as it reads like use this _or_ that variant. But if only the first variant is used instrumenation-pg is not loaded at all.

Reply:
which is exactly as it should be treated, if you want to load only `PostgresInstrumentation` then don't call `registerInstrumentations`, unless you want to have both. I have added 3rd example, now all should be clear

Reply:
Thanks for the update. The second and the third are my favorites.
Still think the first one (load all default plugins but not pg-instrumentation) could be removed but feel free to keep it.